cmake/modules/FindASSIMP.cmake

```diff
@@ -31,7 +31,7 @@ DEAL_II_FIND_LIBRARY(ASSIMP_LIB NAMES assimp
   )
 
 DEAL_II_FIND_PATH(ASSIMP_INC assimp/defs.h
-  HINTS ${ASSIMP_DIR}
+  HINTS ${ASSIMP_DIR} ${ASSIMP_DIR}/include/
```

Comment:
One nit-pick: Can you refactor to:
```
DEAL_II_FIND_PATH(ASSIMP_INC assimp/defs.h
  HINTS ${ASSIMP_DIR}
  PATH_SUFFIXES include
```
?
And quickly check :-)

Reply:
sure

Reply:
done